ql/src/test/results/clientpositive/llap/windowing_udaf.q.out

```diff
@@ -505,7 +505,7 @@ alice brown	25.258749999999996
 alice brown	25.529374999999998
 alice brown	25.63012987012987
 alice brown	26.472439024390237
-alice brown	27.27881720430106
+alice brown	27.100638297872322
```

Comment:
https://github.com/apache/hive/pull/5743/files#diff-3ed1e47d0499aed223f40aaacf7d3b5d17ec6c6e63d3f986209c0c217bcdb513R930
I am seeing references of window function using the `isEqualPrimitive()` in its function call stack. I am assuming this to be the reason why this value is changing.